src/Pages/PeoplePage.tsx

```diff
@@ -0,0 +1,73 @@
+import React, { useEffect, useState } from 'react';
+import { getPeople } from '../api';
+import { Person } from '../types/Person';
+import { PeopleTable } from '../components/PeopleTable';
+import { Loader } from '../components/Loader';
+
+const findParent = (people: Person[], name: string) => {
+  return people.find(person => person.name === name);
```

Comment:
I guess will be more logical to set null if nothing is found, but it's not critical

I mean - undefined tell us that somehow data wasn't set here due to mistake, error or something like that
null - tell us that there shouldn't be anything intentionally